packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1445,15 +1469,34 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 			throw new LoggingError("Change API requires an ID that is a string");
 		}
 
+		// prevent the overwriting of an interval label, it should remain unchanged
+		// once it has been inserted into the collection.
+		if (props !== undefined && props[reservedRangeLabelsKey] !== undefined) {
```

Comment:
```suggestion
		if (props?[reservedRangeLabelsKey] !== undefined) {
```

Reply:
I think I meant to leave a PR comment about this and forgot - I had your suggested change originally, but it gives the error "This condition will always return 'true' since JavaScript compares objects by reference, not value."

Reply:
oh oops.. that's because it's not the right syntax. good linter :). The right syntax is this:

```suggestion
		if (props?.[reservedRangeLabelsKey] !== undefined) {
```